### ISSUE TITLE
[Accessibility]  NVDA does not read as sorted descending and sorted ascending after click on columns.

### ISSUE THREAD

**Issue by ashukum-msft** [angular-ui/ui-grid#7075]
**Issue** - When you click on column header, NVDA does not read as sorted descending and sorted ascending after click on columns. 

**Expected:**
NVDA should read as sorted ascending and sorted descending after click on columns.
**User Impact:**
Screenreader user unable to know that whether it is sorted ascending or descending.

Please find the sample plnkr link below -- 
http://plnkr.co/edit/zMy3uZ5ykITt2pOt0AyX?p=preview
